Dockerfile

```diff
@@ -1,93 +1,48 @@
-FROM ubuntu:18.04
+FROM ubuntu:22.04
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update
+RUN apt-get install --no-install-recommends -y apt-transport-https
+RUN apt-get update
+RUN apt-get install --no-install-recommends -y curl gnupg2 ca-certificates software-properties-common nlohmann-json3-dev
+
+RUN apt-get install --no-install-recommends -y git cmake build-essential gcc-9 g++-9 sqlite3 libsqlite3-dev libssl-dev librdkafka-dev libboost-all-dev libtool libxerces-c-dev libflatbuffers-dev
+RUN update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-11 10
+RUN update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 20
+
+RUN update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-11 10
+RUN update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9 20
+
+RUN add-apt-repository ppa:deadsnakes/ppa
+RUN apt-get install --no-install-recommends -y python3.11-dev
+RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+RUN apt-get install --no-install-recommends -y libjsoncpp-dev libspdlog-dev pigz
+RUN python3.11 get-pip.py
+RUN pip install tensorflow==2.5.3
```

Comment:
Tensorflow version need to be updated to 2.12 or later
https://github.com/tensorflow/tensorflow/releases/tag/v2.12.0.

Just wondering why we are installing python packages here as the GraphSAGE requirements are installed in the jasminegraph docker build. 
https://github.com/miyurud/jasminegraph/pull/176/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R9

Reply:
Need to decide where the python packages are to be installed. I think it's better to keep it on Jasminegraph repo instead of doing it here as the package versions can be updated frequently.
We can remove the `python_libs` folder of this repo

Reply:
> Just wondering why we are installing python packages here as the GraphSAGE requirements are installed in the jasminegraph docker build and the requirements file of it have got the updated version as well.

We have to prevent upgrading the already installed Python packages when building jasminegraph docker image. Installing them here will save time when building the jasminegraph docker image.

Reply:
Lets add python module installation in the JasmineGraph prerequisites image. That will remove the burden of re-downloading all the module content.